Sources/CocoaLumberjack/DDLog.m

```diff
@@ -1135,6 +1135,16 @@ - (id)copyWithZone:(NSZone * __attribute__((unused)))zone {
     return newMessage;
 }
 
+- (void)setRepresentedObject:(id)representedObject {
```

Comment:
This means, the backing storage is still `_tag`. Like I mentioned before, I'd prefer to have the internal code base using the new and recommended way, whilst having the `tag` as custom getter/setter implementation to keep the backwards compatibility.

This would also solve the problem where `DD_LEGACY_MESSAGE_TAG` is set to `0` and suddenly there is no backing storage for `tag` anymore, which would cause these getter/setter implementations to fail.